---
title: "References"
date: "`r Sys.Date()`"
output:
  bookdown::html_document2: 
    toc: FALSE
    toc_float: TRUE
    theme: united
link-citations: yes
fontsize: 12pt
urlcolor: blue
csl: AmJBot.csl
bibliography: References.bib
editor_options: 
  markdown: 
    wrap: sentence
---
1.	Kwon, J.M., Goate, A.M. The candidate gene approach. Alcohol Res Health, 2000, 24, 164-168.
2.	Moore, S.R. Commentary: What Is the Case for Candidate Gene Approaches in the Era of High-Throughput Genomics? A Re-sponse to Border and Keller (2017). Journal of Child Psychology and Psychiatry 2017, 58, 331–334, doi:10.1111/jcpp.12697.
3.	Tam, V.; Patel, N.; Turcotte, M.; Bossé, Y.; Paré, G.; Meyre, D. Benefits and Limitations of Genome-Wide Association Studies. Nature Reviews Genetics 2019, 20, 467–484, doi:10.1038/s41576-019-0127-1.
4.	Idaho GEM3 Genes by Environment Available online: https://www.idahogem3.org/ (accessed on 18 December 2020).
5.	Luikart, G.; England, P.R.; Tallmon, D.; Jordan, S.; Taberlet, P. The Power and Promise of Population Genomics: From Geno-typing to Genome Typing. Nature Reviews Genetics 2003, 4, 981–994, doi:10.1038/nrg1226.
6.	Ellegren, H. Genome Sequencing and Population Genomics in Non-Model Organisms. Trends in Ecology & Evolution 2014, 29, 51–63, doi:10.1016/j.tree.2013.09.008.
7.	Tao, Y.; Cai, C.; Cohen, W.W.; Lu, X. From genome to phenome: Predicting multiple cancer phenotypes based on somatic ge-nomic alterations via the genomic impact transformer. In Biocomputing 2020; WORLD SCIENTIFIC, 2019; pp. 79–90 ISBN 9789811215629.
8.	London, S.J.; Romieu, I. Gene by Environment Interaction in Asthma. Annual Review of Public Health 2009, 30, 55–80, doi:10.1146/annurev.publhealth.031308.100151.
9.	Lendenmann, M.H.; Croll, D.; Palma-Guerrero, J.; Stewart, E.L.; McDonald, B.A. QTL Mapping of Temperature Sensitivity Reveals Candidate Genes for Thermal Adaptation and Growth Morphology in the Plant Pathogenic Fungus Zymoseptoria Tritici. Heredity 2016, 116, 384–394, doi:10.1038/hdy.2015.111.
10.	Russell, J.J.; Theriot, J.A.; Sood, P.; Marshall, W.F.; Landweber, L.F.; Fritz-Laylin, L.; Polka, J.K.; Oliferenko, S.; Gerbich, T.; Gladfelter, A.; et al. Non-Model Model Organisms. BMC Biology 2017, 15, 55, doi:10.1186/s12915-017-0391-5.
11.	Galla, S.J.; Forsdick, N.J.; Brown, L.; Hoeppner, M.P.; Knapp, M.; Maloney, R.F.; Moraga, R.; Santure, A.W.; Steeves, T.E. Ref-erence Genomes from Distantly Related Species Can Be Used for Discovery of Single Nucleotide Polymorphisms to Inform Conservation Management. Genes 2019, 10, 9, doi:10.3390/genes10010009.
12.	Burnett, K.G.; Durica, D.S.; Mykles, D.L.; Stillman, J.H.; Schmidt, C. Recommendations for Advancing Genome to Phenome Research in Non-Model Organisms. Integr Comp Biol, doi:10.1093/icb/icaa059.
13.	Zargar, S.M.; Raatz, B.; Sonah, H.; MuslimaNazir; Bhat, J.A.; Dar, Z.A.; Agrawal, G.K.; Rakwal, R. Recent Advances in Mo-lecular Marker Techniques: Insight into QTL Mapping, GWAS and Genomic Selection in Plants. J. Crop Sci. Biotechnol. 2015, 18, 293–308, doi:10.1007/s12892-015-0037-5.
14.	Egmond, M.E. van; Lugtenberg, C.H.A.; Brouwer, O.F.; Contarino, M.F.; Fung, V.S.C.; Heiner‐Fokkema, M.R.; Hilten, J.J. van; Hout, A.H. van der; Peall, K.J.; Sinke, R.J.; et al. A Post Hoc Study on Gene Panel Analysis for the Diagnosis of Dystonia. Movement Disorders 2017, 32, 569–575, doi:https://doi.org/10.1002/mds.26937.
15.	Zhu, M.; Zhao, S. Candidate Gene Identification Approach: Progress and Challenges. Int J Biol Sci 2007, 3, 420–427.
16.	Border, R.; Keller, M.C. Commentary: Fundamental Problems with Candidate Gene-by-Environment Interaction Studies – Reflections on Moore and Thoemmes (2016). Journal of Child Psychology and Psychiatry 2017, 58, 328–330, doi:https://doi.org/10.1111/jcpp.12669.
17.	Bakshi, R.K.; Kaur, N.; Kaur, R.; Kaur, G. Opinion Mining and Sentiment Analysis. In Proceedings of the 2016 3rd Interna-tional Conference on Computing for Sustainable Global Development (INDIACom); March 2016; pp. 452–455.
18.	Aria, M.; Cuccurullo, C. Bibliometrix: An R-Tool for Comprehensive Science Mapping Analysis. Journal of Informetrics 2017, 11, 959–975, doi:10.1016/j.joi.2017.08.007.
19.	R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Aus-tria 2019, URL https://www.R-project.org/.
20.	Wickham, H.; Hester, J.; Chang, W.; RStudio; R), R.C. team (Some namespace and vignette code extracted from base Devtools: Tools to Make Developing R Packages Easier; 2020.
21.	Epskamp, S., Cramer, A.O.J., Waldorp, L.J., Schmittmann, V.D., Borsboom, D. qgraph: Network Visualizations of Relation-ships in Psychometric Data. Journal of Statistical Software, 2012, 48, 1-18.
22.	Dormann, C.F., Gruber, B., Freund, J. Introducing the bipartite package: analysing ecological networks. R News 2008, 8/2, 8-11.
23.	Roberts, R.J. PubMed Central: The GenBank of the Published Literature. PNAS 2001, 98, 381–382, doi:10.1073/pnas.98.2.381.
24.	Burnham, J.F. Scopus Database: A Review. Biomedical Digital Libraries 2006, 3, 1, doi:10.1186/1742-5581-3-1.
25.	Harzing, A.-W.; Alakangas, S. Google Scholar, Scopus and the Web of Science: A Longitudinal and Cross-Disciplinary Comparison. Scientometrics 2016, 106, 787–804, doi:10.1007/s11192-015-1798-9. 
26.	Kovalchik, S. RISmed: Download Content from NCBI Databases. R package version 2.2 2020. https://CRAN.R-project.org/package=RISmed 
27.	Fantini, D. easyPubMed: Search and Retrieve Scientific Publication Records from PubMed. R package version 2.13 2019. https://CRAN.R-project.org/package=easyPubMed
28.	Selivanov, D., Bickel, M., Wang, Q.  text2vec: Modern Text Mining Framework for R. R package version 0.6 2020.
29.	Csardi G., Nepusz T. The igraph software package for complex network research. InterJournal, Complex Systems, 2006, 1695.
30.	Bairoch, A., Boeckmann, B. The SWISS-PROT protein sequence data bank. Nucleic Acids Research, 1991, 19, 2247–2249.
31.	Global Biodiversity Information Facility. GBIF Memorandum of Understanding 2010 https://doi.org/10.15468/doc.ajz7-qt28
32.	Chamberlain, S., Szocs, E. taxize – taxonomic search and retrieval in R. F1000Research. 2013, 2:191. URL: http://f1000research.com/articles/2-191/v2.
33.	Cayuela, L.; Macarro, I.; Stein, A.; Oksanen, J. Taxonstand: Taxonomic Standardization of Plant Species Names; 2019;
34.	Missouri Botanical Gardens. http://www.mobot.org/MOBOT/Research/APweb/top/glossarya_h.html
35.	Collins A.; Speer, B.; Waggoner, B.; Whitney, C.; Rieboldt, S. UC Museum of Paleontology Glossary: Zoology Available online: https://ucmp.berkeley.edu/glossary/augloss.html (accessed on 21 December 2020).
36.	Ellis, D. Glossary of Mycological Terms | Mycology Online Available online: https://mycology.adelaide.edu.au/glossary/ (accessed on 21 December 2020).
37.	Estravis-Barcala, M.; Mattera, M.G.; Soliani, C.; Bellora, N.; Opgenoorth, L.; Heer, K.; Arana, M.V. Molecular Bases of Re-sponses to Abiotic Stress in Trees. Journal of Experimental Botany 2020, 71, 3765–3779, doi:10.1093/jxb/erz532.
38.	Jenks, M.A.; Hasegawa, P.M. Plant Abiotic Stress 2005, Blackwell publishing. New Jersey.
39.	Haak, D.C.; Fukao, T.; Grene, R.; Hua, Z.; Ivanov, R.; Perrella, G.; Li, S. Multilevel Regulation of Abiotic Stress Responses in Plants. Front. Plant Sci. 2017, 8, doi:10.3389/fpls.2017.01564.
40.	Striberny, B.; Melton, A.E.; Schwacke, R.; Krause, K.; Fischer, K.; Goertzen, L.R.; Rashotte, A.M. Cytokinin Response Factor 5 Has Transcriptional Activity Governed by Its C-Terminal Domain. Plant Signaling & Behavior 2017, 12, e1276684, doi:10.1080/15592324.2016.1276684.
41.	Menezes-Benavente, L.; Teixeira, F.K.; Alvim Kamei, C.L.; Margis-Pinheiro, M. Salt Stress Induces Altered Expression of Genes Encoding Antioxidant Enzymes in Seedlings of a Brazilian Indica Rice (Oryza Sativa L.). Plant Science 2004, 166, 323–331, doi:10.1016/j.plantsci.2003.10.001.
