<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />



<meta name="date" content="2021-03-19" />

<title>References</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/united.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">G2PMineR</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="ReproducibleCode.html">Reproducible Pipeline</a>
</li>
<li>
  <a href="Figures.html">Vignette Figures</a>
</li>
<li>
  <a href="Tables.html">Vignette Objects</a>
</li>
<li>
  <a href="Lexicon.html">Function Lexicon</a>
</li>
<li>
  <a href="References.html">References</a>
</li>
<li>
  <a href="Funding.html">Funding</a>
</li>
<li>
  <a href="License.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">References</h1>
<h4 class="date">2021-03-19</h4>

</div>


<ol style="list-style-type: decimal">
<li>Kwon, J.M., Goate, A.M. The candidate gene approach. Alcohol Res Health, 2000, 24, 164-168.</li>
<li>Moore, S.R. Commentary: What Is the Case for Candidate Gene Approaches in the Era of High-Throughput Genomics? A Re-sponse to Border and Keller (2017). Journal of Child Psychology and Psychiatry 2017, 58, 331–334, <a href="doi:10.1111/jcpp.12697" class="uri">doi:10.1111/jcpp.12697</a>.</li>
<li>Tam, V.; Patel, N.; Turcotte, M.; Bossé, Y.; Paré, G.; Meyre, D. Benefits and Limitations of Genome-Wide Association Studies. Nature Reviews Genetics 2019, 20, 467–484, <a href="doi:10.1038/s41576-019-0127-1" class="uri">doi:10.1038/s41576-019-0127-1</a>.</li>
<li>Idaho GEM3 Genes by Environment Available online: <a href="https://www.idahogem3.org/" class="uri">https://www.idahogem3.org/</a> (accessed on 18 December 2020).</li>
<li>Luikart, G.; England, P.R.; Tallmon, D.; Jordan, S.; Taberlet, P. The Power and Promise of Population Genomics: From Geno-typing to Genome Typing. Nature Reviews Genetics 2003, 4, 981–994, <a href="doi:10.1038/nrg1226" class="uri">doi:10.1038/nrg1226</a>.</li>
<li>Ellegren, H. Genome Sequencing and Population Genomics in Non-Model Organisms. Trends in Ecology &amp; Evolution 2014, 29, 51–63, <a href="doi:10.1016/j.tree.2013.09.008" class="uri">doi:10.1016/j.tree.2013.09.008</a>.</li>
<li>Tao, Y.; Cai, C.; Cohen, W.W.; Lu, X. From genome to phenome: Predicting multiple cancer phenotypes based on somatic ge-nomic alterations via the genomic impact transformer. In Biocomputing 2020; WORLD SCIENTIFIC, 2019; pp. 79–90 ISBN 9789811215629.</li>
<li>London, S.J.; Romieu, I. Gene by Environment Interaction in Asthma. Annual Review of Public Health 2009, 30, 55–80, <a href="doi:10.1146/annurev.publhealth.031308.100151" class="uri">doi:10.1146/annurev.publhealth.031308.100151</a>.</li>
<li>Lendenmann, M.H.; Croll, D.; Palma-Guerrero, J.; Stewart, E.L.; McDonald, B.A. QTL Mapping of Temperature Sensitivity Reveals Candidate Genes for Thermal Adaptation and Growth Morphology in the Plant Pathogenic Fungus Zymoseptoria Tritici. Heredity 2016, 116, 384–394, <a href="doi:10.1038/hdy.2015.111" class="uri">doi:10.1038/hdy.2015.111</a>.</li>
<li>Russell, J.J.; Theriot, J.A.; Sood, P.; Marshall, W.F.; Landweber, L.F.; Fritz-Laylin, L.; Polka, J.K.; Oliferenko, S.; Gerbich, T.; Gladfelter, A.; et al. Non-Model Model Organisms. BMC Biology 2017, 15, 55, <a href="doi:10.1186/s12915-017-0391-5" class="uri">doi:10.1186/s12915-017-0391-5</a>.</li>
<li>Galla, S.J.; Forsdick, N.J.; Brown, L.; Hoeppner, M.P.; Knapp, M.; Maloney, R.F.; Moraga, R.; Santure, A.W.; Steeves, T.E. Ref-erence Genomes from Distantly Related Species Can Be Used for Discovery of Single Nucleotide Polymorphisms to Inform Conservation Management. Genes 2019, 10, 9, <a href="doi:10.3390/genes10010009" class="uri">doi:10.3390/genes10010009</a>.</li>
<li>Burnett, K.G.; Durica, D.S.; Mykles, D.L.; Stillman, J.H.; Schmidt, C. Recommendations for Advancing Genome to Phenome Research in Non-Model Organisms. Integr Comp Biol, <a href="doi:10.1093/icb/icaa059" class="uri">doi:10.1093/icb/icaa059</a>.</li>
<li>Zargar, S.M.; Raatz, B.; Sonah, H.; MuslimaNazir; Bhat, J.A.; Dar, Z.A.; Agrawal, G.K.; Rakwal, R. Recent Advances in Mo-lecular Marker Techniques: Insight into QTL Mapping, GWAS and Genomic Selection in Plants. J. Crop Sci. Biotechnol. 2015, 18, 293–308, <a href="doi:10.1007/s12892-015-0037-5" class="uri">doi:10.1007/s12892-015-0037-5</a>.</li>
<li>Egmond, M.E. van; Lugtenberg, C.H.A.; Brouwer, O.F.; Contarino, M.F.; Fung, V.S.C.; Heiner‐Fokkema, M.R.; Hilten, J.J. van; Hout, A.H. van der; Peall, K.J.; Sinke, R.J.; et al. A Post Hoc Study on Gene Panel Analysis for the Diagnosis of Dystonia. Movement Disorders 2017, 32, 569–575, <a href="doi:https://doi.org/10.1002/mds.26937" class="uri">doi:https://doi.org/10.1002/mds.26937</a>.</li>
<li>Zhu, M.; Zhao, S. Candidate Gene Identification Approach: Progress and Challenges. Int J Biol Sci 2007, 3, 420–427.</li>
<li>Border, R.; Keller, M.C. Commentary: Fundamental Problems with Candidate Gene-by-Environment Interaction Studies – Reflections on Moore and Thoemmes (2016). Journal of Child Psychology and Psychiatry 2017, 58, 328–330, <a href="doi:https://doi.org/10.1111/jcpp.12669" class="uri">doi:https://doi.org/10.1111/jcpp.12669</a>.</li>
<li>Bakshi, R.K.; Kaur, N.; Kaur, R.; Kaur, G. Opinion Mining and Sentiment Analysis. In Proceedings of the 2016 3rd Interna-tional Conference on Computing for Sustainable Global Development (INDIACom); March 2016; pp. 452–455.</li>
<li>Aria, M.; Cuccurullo, C. Bibliometrix: An R-Tool for Comprehensive Science Mapping Analysis. Journal of Informetrics 2017, 11, 959–975, <a href="doi:10.1016/j.joi.2017.08.007" class="uri">doi:10.1016/j.joi.2017.08.007</a>.</li>
<li>R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Aus-tria 2019, URL <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a>.</li>
<li>Wickham, H.; Hester, J.; Chang, W.; RStudio; R), R.C. team (Some namespace and vignette code extracted from base Devtools: Tools to Make Developing R Packages Easier; 2020.</li>
<li>Epskamp, S., Cramer, A.O.J., Waldorp, L.J., Schmittmann, V.D., Borsboom, D. qgraph: Network Visualizations of Relation-ships in Psychometric Data. Journal of Statistical Software, 2012, 48, 1-18.</li>
<li>Dormann, C.F., Gruber, B., Freund, J. Introducing the bipartite package: analysing ecological networks. R News 2008, 8/2, 8-11.</li>
<li>Roberts, R.J. PubMed Central: The GenBank of the Published Literature. PNAS 2001, 98, 381–382, <a href="doi:10.1073/pnas.98.2.381" class="uri">doi:10.1073/pnas.98.2.381</a>.</li>
<li>Burnham, J.F. Scopus Database: A Review. Biomedical Digital Libraries 2006, 3, 1, <a href="doi:10.1186/1742-5581-3-1" class="uri">doi:10.1186/1742-5581-3-1</a>.</li>
<li>Harzing, A.-W.; Alakangas, S. Google Scholar, Scopus and the Web of Science: A Longitudinal and Cross-Disciplinary Comparison. Scientometrics 2016, 106, 787–804, <a href="doi:10.1007/s11192-015-1798-9" class="uri">doi:10.1007/s11192-015-1798-9</a>.</li>
<li>Kovalchik, S. RISmed: Download Content from NCBI Databases. R package version 2.2 2020. <a href="https://CRAN.R-project.org/package=RISmed" class="uri">https://CRAN.R-project.org/package=RISmed</a></li>
<li>Fantini, D. easyPubMed: Search and Retrieve Scientific Publication Records from PubMed. R package version 2.13 2019. <a href="https://CRAN.R-project.org/package=easyPubMed" class="uri">https://CRAN.R-project.org/package=easyPubMed</a></li>
<li>Selivanov, D., Bickel, M., Wang, Q. text2vec: Modern Text Mining Framework for R. R package version 0.6 2020.</li>
<li>Csardi G., Nepusz T. The igraph software package for complex network research. InterJournal, Complex Systems, 2006, 1695.</li>
<li>Bairoch, A., Boeckmann, B. The SWISS-PROT protein sequence data bank. Nucleic Acids Research, 1991, 19, 2247–2249.</li>
<li>Global Biodiversity Information Facility. GBIF Memorandum of Understanding 2010 <a href="https://doi.org/10.15468/doc.ajz7-qt28" class="uri">https://doi.org/10.15468/doc.ajz7-qt28</a></li>
<li>Chamberlain, S., Szocs, E. taxize – taxonomic search and retrieval in R. F1000Research. 2013, 2:191. URL: <a href="http://f1000research.com/articles/2-191/v2" class="uri">http://f1000research.com/articles/2-191/v2</a>.</li>
<li>Cayuela, L.; Macarro, I.; Stein, A.; Oksanen, J. Taxonstand: Taxonomic Standardization of Plant Species Names; 2019;</li>
<li>Missouri Botanical Gardens. <a href="http://www.mobot.org/MOBOT/Research/APweb/top/glossarya_h.html" class="uri">http://www.mobot.org/MOBOT/Research/APweb/top/glossarya_h.html</a></li>
<li>Collins A.; Speer, B.; Waggoner, B.; Whitney, C.; Rieboldt, S. UC Museum of Paleontology Glossary: Zoology Available online: <a href="https://ucmp.berkeley.edu/glossary/augloss.html" class="uri">https://ucmp.berkeley.edu/glossary/augloss.html</a> (accessed on 21 December 2020).</li>
<li>Ellis, D. Glossary of Mycological Terms | Mycology Online Available online: <a href="https://mycology.adelaide.edu.au/glossary/" class="uri">https://mycology.adelaide.edu.au/glossary/</a> (accessed on 21 December 2020).</li>
<li>Estravis-Barcala, M.; Mattera, M.G.; Soliani, C.; Bellora, N.; Opgenoorth, L.; Heer, K.; Arana, M.V. Molecular Bases of Re-sponses to Abiotic Stress in Trees. Journal of Experimental Botany 2020, 71, 3765–3779, <a href="doi:10.1093/jxb/erz532" class="uri">doi:10.1093/jxb/erz532</a>.</li>
<li>Jenks, M.A.; Hasegawa, P.M. Plant Abiotic Stress 2005, Blackwell publishing. New Jersey.</li>
<li>Haak, D.C.; Fukao, T.; Grene, R.; Hua, Z.; Ivanov, R.; Perrella, G.; Li, S. Multilevel Regulation of Abiotic Stress Responses in Plants. Front. Plant Sci. 2017, 8, <a href="doi:10.3389/fpls.2017.01564" class="uri">doi:10.3389/fpls.2017.01564</a>.</li>
<li>Striberny, B.; Melton, A.E.; Schwacke, R.; Krause, K.; Fischer, K.; Goertzen, L.R.; Rashotte, A.M. Cytokinin Response Factor 5 Has Transcriptional Activity Governed by Its C-Terminal Domain. Plant Signaling &amp; Behavior 2017, 12, e1276684, <a href="doi:10.1080/15592324.2016.1276684" class="uri">doi:10.1080/15592324.2016.1276684</a>.</li>
<li>Menezes-Benavente, L.; Teixeira, F.K.; Alvim Kamei, C.L.; Margis-Pinheiro, M. Salt Stress Induces Altered Expression of Genes Encoding Antioxidant Enzymes in Seedlings of a Brazilian Indica Rice (Oryza Sativa L.). Plant Science 2004, 166, 323–331, <a href="doi:10.1016/j.plantsci.2003.10.001" class="uri">doi:10.1016/j.plantsci.2003.10.001</a>.</li>
</ol>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
